q2_fmt/plugin_setup.py

```diff
@@ -342,28 +310,28 @@
         'sampling_depth': sampling_depth
     },
     output_descriptions={
-        'peds_dists': peds_dists
+        'prdf_dists': pedf_dists
     },
-    name='Porportional Engraftment of Donor Strains per feature',
+    name='Porportion of Recipient with Donor Feature',
```

Comment:
```suggestion
    name='Porportion of Recipients with Donor Feature',
```
Not sure if this should be pluralized? Ignore if not